language-support/hs/bindings/test/DA/Ledger/Sandbox.hs

```diff
@@ -0,0 +1,123 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Sandbox ( -- Run a sandbox for testing on a dynamically selected port
+    SandboxSpec(..),
+    Sandbox(port),
+    startSandbox,
+    shutdownSandbox,
+    withSandbox
+    ) where
+
+import           Control.Concurrent.Async (race)
+import           Control.Monad            (when)
```

Comment:
No need to give an export list for this and many of the things below.

Reply:
I read a suggestion on Hackage that imports should take either the form: `import qualified M` or `import M(...)`, so it's clear where the definition for every identifier is coming from.

As a Haskell Noob, I like very much being this explicit, especially while I am still finding my way around the standard libraries.

Of course, I accept that some standard definitions (`when` ?) are so pervasive that perhaps the rule doesn't need to apply to them.